### PR TITLE
feat(TokenKeyName): Set a configurable token key name

### PR DESCRIPTION
**Pull request by ElRochito**

### CODE REVIEW

File: config/config.php [hunk 1]
```diff
@@ -27,6 +27,17 @@
...
+    /*
+    |--------------------------------------------------------------------------
+    | JWT Token Key Name
+    |--------------------------------------------------------------------------
+    |
+    | You can set a custom token key name in your .env file
+    |
+    */
+
+    'token_key_name' => env('JWT_TOKEN_KEY_NAME', 'token'),
```
Comment (tymondesigns) [inline config/config.php]: I don't think this needs to be configurable at the env level

**Review by tymondesigns — CHANGES_REQUESTED** [review]

File: config/config.php (hunk 1 above)
Comment (bastien-phi) [reply config/config.php]: Maybe just `'token_key_name' => 'token',` should be better ?

**Review by ElRochito — COMMENTED** [review]
I've changed config/config.php to set a token name directly in config/jwt.php

**Review by ElRochito — COMMENTED** [review]
Configurable in config.jwt.php instead of env file

**Review by ElRochito — COMMENTED** [review]
I've updated my code

**Comment by tymondesigns** [comment]
Thank you for your contribution but looking at this again, I don't think it's necessary.

**Pull request closed without merging**